File: 6th Semester/4 programs 4 languages assignment/ex1.java
import java.util.Scanner;

public class ex1
{
	public static void main(String[] args) {
	    Scanner scanner = new Scanner(System.in);
		System.out.print("Qual a velocidade do carro: ");
		float velocidade = scanner.nextFloat();
		if (velocidade < 0) velocidade = -velocidade;
		if (velocidade <= 80) {
		    System.out.println("Velocidade abaixo de 80km/h. Você não foi multado (>ω<)");
		    return;
		}
		System.out.println("Velocidade acima do permitido!!!!! (# ﾟДﾟ)");
		System.out.format("Você pagará uma multa de R$%.2f", (velocidade-80)*5);
	}
}

File: 6th Semester/4 programs 4 languages assignment/ex1.cs
using System;
  
public class ex1 {

    static public float leitor(string message) {
        float number;
        while (true)
        {
            Console.Write(message);
            if (float.TryParse(Console.ReadLine(), out number)) return number;
            Console.WriteLine("Valor inválido. Tente novamente.");
        }
    }
  
    static public void Main()
    {

        //Console.Write("Qual a velocidade do carro:  ");
        float velocidade = leitor("Qual a velocidade do carro: ");
        
        if (velocidade < 80) {
            Console.WriteLine("Velocidade abaixo de 80km/h. Você não foi multado (>ω<)");
            return;
        }
        float multa = (velocidade - 80) * 5;
        Console.WriteLine("Velocidade acima do permitido!!!!! (# ﾟДﾟ)");
        Console.WriteLine(string.Format("Você pagará uma multa de R${0:0,00}", multa));
    }
}